pom.xml

```diff
@@ -258,19 +271,15 @@
 				<artifactId>maven-source-plugin</artifactId>
 			</plugin>
 			<plugin>
-				<!-- 
-					Use the Nexus Staging plugin as a full replacement for the standard
-					Maven Deploy plugin.
-					See https://github.com/sonatype/nexus-maven-plugins/tree/master/staging/maven-plugin
-					why this makes sense :-)
-					We can control whether we want to deploy to the Eclipse repo or Maven Central
-					by a combination of the version being a SNAPHOT or release version and property
-					skipStaging=true/false.
-					In any case we can take advantage of the plugin's "deferred deploy" feature which
-					makes sure that all artifacts of a multi-module project are deployed as a whole
-					at the end of the build process instead of deploying each module's artifacts
-					individually as part of building the module.
-				 -->
+				<!-- Use the Nexus Staging plugin as a full replacement for the standard 
```

Comment:
Hmm,  this looks just that a lot of stuff is not formatted according the formatter ;-(.
This make really a lot unnecessary work.